### PR TITLE
just some docker formating changes

### PR DESCRIPTION
**Pull request by bashbang**
# Description

This PR includes the following proposed change(s):

Updated some docker formatting and set perms on keycloak create user script.
Removed Acquire::Check-Valid-Until line - this seems to be a workaround.  Need to test this build on a windows box.  Works without the Acquire on a *nix box. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Run locally

## Does the change impact or break the Docker build?

- [X] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [X] Yes
- [ ] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .settings/keycloak/Dockerfile [hunk 1]
```diff
@@ -6,14 +6,17 @@ ARG DOCKERIZE_VERSION=v0.6.1
 
 ENV KEYCLOAK_URL=$KEYCLOAK_URL
 
+# TODO: I didn't need this for my build - this needs to be verified on a windows box?
```
Comment (akroon3r) [inline .settings/keycloak/Dockerfile:9]: Should we start creating JIRA tasks on our board for TODOs?

Comment (bashbang) [reply .settings/keycloak/Dockerfile:9]: Is there a way to automate that?

Comment (akroon3r) [reply .settings/keycloak/Dockerfile:9]: given that it's government JIRA, most likely not lol

**Review by akroon3r — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -166,6 +166,9 @@ services:
     networks:
       - fisu-net
 
+  # #############################################################################################
+  # ###                                        KEYCLOAK Config                                ###
+  # #############################################################################################
```
Comment (alexjoybc) [inline docker-compose.yml:171]: thanks ;-)

File: .settings/keycloak/Dockerfile (hunk 1 above)
Comment (alexjoybc) [reply .settings/keycloak/Dockerfile:9]: nope ;-( no automation

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**